autoinstall/README.md

```diff
@@ -0,0 +1,16 @@
+# Autoinstall
+By using autoinstall we could automate the installer like what `presed` is used for `d-i`.
+
+## Usage
+1. Upload your `user-data` to a www server. You may use `basic.yaml` as an example. Rename `basic.yaml` and put it 
+   into a www server to serve it.
```

Comment:
Do you have to rename it? If so, what should it be renamed to?

Reply:
Yes, `basic.yaml` should be renamed to `user-data`, which is the spec of `cloud-init`.

Reply:
I should have used statement like `Rename basic.yaml to be user-data` to make it clear. I will update it.